recipe/meta.yaml

```diff
@@ -104,6 +106,10 @@ requirements:
     #- vs2015_runtime    # [win]
```

Comment:
Should we be including `gdbm` at runtime?

Reply:
GOod question. My vote is no to keep GPL library out of the core dependencies.

Reply:
GOod question. My vote is no to keep GPL library out of the core dependencies.

Reply:
An other reason would be to keep the python package lean. I'm sure there are many other options that could be added, but not everybody might need them. Since it wasn't really creating issues before, I assume that it is OK as is.